Automattic/wc-calypso-bridge
# Make the domain purchase note appear sooner and increased visibility

### Changes proposed in this Pull Request:



This PR makes the domain purchase note show up within 1 hour from store creation, instead of 1 day. It also makes the note unactioned and conditionally marks the note as unread for users who just upgraded to a paid plan, to increase visibility and, hopefully, conversion.

Closes #1365 






### How to test the changes in this Pull Request:




1. Create a new free trial Woo Express site and prepare it for development.
2. Upload a new Calypso Bridge version via `woa`.
3. Wait for one hour for the CRON to run or run the following command in the `cli`: `wp cron event run wc_calypso_bridge_hourly`.
4. In the My Home page, ensure that a note to choose a domain shows up. 
5. Click the "Choose a domain" button -- it will open up a new tab. 
6. Close the new tab without choosing a domain. 
7. Refresh the My Home page and ensure that the note about choosing a domain is still there. 
8. Upgrade to a paid plan (Essential or Performance). 
9. Wait for one hour for the CRON to run or run the following command in the `cli`: `wp cron event run wc_calypso_bridge_hourly`.
10. Refresh the My Home page and ensure that the note appears unread. 
11. Either purchase a domain or dismiss the note. 
12. Wait for one hour for the CRON to run or run the following command in the `cli`: `wp cron event run wc_calypso_bridge_hourly`.
13. Refresh the My Home page and ensure that the note is not re-added. 



### Other information:




### FOR PR REVIEWER ONLY:
